### DISCUSSION TITLE
Step ca docker command not mounting correct volume path

### DISCUSSION THREAD

**Discussion by amit12cool** [smallstep/certificates#801] [Q&A]
### Subject of the issue
Describe your issue here.
When I run the below docker command 
```
docker run -d -v  step:/home/step/edgedevices \
    -p 9000:9000 \
    -e "DOCKER_STEPCA_INIT_NAME=Veli-CA-For-Edge-Devices" \
    -e "DOCKER_STEPCA_INIT_DNS_NAMES=localhost,$(hostname -f)" \
    -e "DOCKER_STEPCA_INIT_CONTEXT=edgedevices" \
    smallstep/step-ca
```

And login to the container, there is no `edgeDevices` directory rather there is `/home/step`  directory present.

### Your environment
* OS -  Ubuntu 
* Version - 18.04

### Steps to reproduce
Run above docker command

### Expected behaviour
Correct volume path should be mounted 

### Actual behaviour
`/home/step` volume is mounted always

Also let me know how can i generate CA_FINGERPRINT if I have 2 docker container running i.e. I have 2 CA's running in docker containers. Please let me know the docker command.

**Comment by tashian** [comment]
Hi! Thanks for sharing this.

The `step-ca` Docker image doesn't support all of the flags that one could pass to `step ca init` as `DOCKER_STEPCA_INIT_` variables. It only supports those variables listed in the `README` on Docker Hub.

However, the `step` command itself accepts flags passed via `STEP_<flag name>` env variables. So, you could try passing `STEP_CONTEXT=edgedevices`.

If that doesn't work for you, I'd suggest entering the container and running the init command directly.
